src/store/region-context.tsx

```diff
@@ -84,17 +84,21 @@ export const RegionProvider = ({ children }: RegionProviderProps) => {
         const is_dev = isLocalhost() || isTestlink()
         if (website_status) {
             const { clients_country, p2p_config } = website_status
+            if (is_p2p_enabled()) {
+                setP2PAllowedCountry(true)
+                setP2PLoading(false)
+            } else if ('p2p_config' in website_status && p2p_config) {
+                setP2PAllowedCountry(true)
+                setP2PLoading(false)
```

Comment:
hi @Nuzhy-Deriv 
here in this if & else if  both condition have same  ( setP2PAllowedCountry(true)
                setP2PLoading(false)) content possible to simplify this ?